Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.h

```diff
@@ -97,6 +97,7 @@ WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
 
 @property (readonly, nonatomic) BOOL hasBackgroundContent;
 @property (readonly, nonatomic) BOOL backgroundContentIsPersistent;
+@property (readonly, nonatomic) BOOL backgroundContentUsesModules;
```

Comment:
This will be the public API header, and I don't think we need to expose this as API. You can move to `_WKWebExtensionPrivate.h` to use it in testing. Prefix the property name with `_` in the private header.

Reply:
Renamed to _backgroundContentUsesModules and moved it to the private header!